components/Nav.vue

```diff
@@ -0,0 +1,336 @@
+<template>
+  <nav>
+  <div class="hamburger-menu">
+    <input id="menu__toggle" type="checkbox" />
+    <label class="menu__btn" for="menu__toggle">
+      <span></span>
+    </label>
+
+    <ul class="nav-links" > 
+        <li class="pages"><a href="#">Home</a></li>
+        <li class="pages"><a href="#">Shop</a></li>
+        <li class="pages"><a href="#">About Us</a></li>
+        <li class="get_it"><a href="#"><img src="get_it.png"><span>GetIt</span></a></li> 
+        <li class="pages"><a href="#" class="sign_in">Sign In</a></li>
```

Comment:
can you please see if we can have a better naming of `pages` like this is very generic ?

Reply:
It is good to use nav-link as all are links.

Reply:
Ok , I'll change it to nav-link